community/portworx/values.yaml

```diff
@@ -40,10 +41,7 @@ etcd:
   ca: none                            # Location of CA file for ETCD authentication. Should be /path/to/server.ca
   cert: none                          # Location of certificate for ETCD authentication. Should be /path/to/server.crt
   key: none                           # Location of certificate key for ETCD authentication Should be /path/to/servery.key
-consul:
-  token: none                          # ACL token value used for Consul authentication. (example: 398073a8-5091-4d9c-871a-bbbeb030d1f6)
-imageVersion: 2.0.3.6                  # Version of the PX Image.
-deploymentType: oci                     # accepts "oci" or "docker"
+imageVersion: 2.1.2                  # Version of the PX Image.
```

Comment:
@disrani-px : Is this image version suitable for both openshift and non-openshift based IKS clusters we have in IBM?

Reply:
Yes this will work for both